pkg/machines/libvirt-xml-update.js

```diff
@@ -0,0 +1,325 @@
+import { getElem, getSingleOptionalElem } from './libvirt-common.js';
```

Comment:
Makes total sense.

Reply:
yes, I usually try to remember to name component files with camelcase, but helpers and other non react files don't have any rule. I don't have a preference here.

Reply:
Ah hmm, xmlCreator.js contains functions which contains part of the domain XML. We use them with API functions which take XML and update resources in that way. 
but the libvirt-xml-update function adjust returns always the whole XML of the domain changed.

It's slightly different.

Reply:
So since we decided to start using virt-xml I think this file (xmlCreator) will be deprecated soon.
Therefore I will not try to  merge it with something else now.